### PR TITLE
add subitem for the combined districts layer

### PR DESCRIPTION
**Pull request by rkelson**
change wording for color 4 house districts
remove Political.

### CODE REVIEW

File: data/political/2022-2032-house-senate-congressional-districts/index.html [hunk 1]
```diff
@@ -143,3 +151,22 @@ <h3 id="UnitedStatesCongressionalDistricts">United States Congressional District
...
   </div>
 </div>
+
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="CombinedDistricts">Combined Political Districts 2022 to 2032</h3>
+    <div class="grid__col grid__col--12-of-12 package-content">
+      <img class="inline-text-left outline" src="{% link images/HouseSmall2.png %}" alt="StateHouseNew"
+        loading="lazy" />
+      <p>Utah District Combination Areas 2022 is comprised of all 4 politcal districts including the House, Senate, U.S. Congressional, and School Board districs.</p>
```
Comment (steveoh) [inline data/political/2022-2032-house-senate-congressional-districts/index.html:161]:
```suggestion
      <p>Utah District Combination Areas 2022 to 2032 is comprised of all 4 political districts including the House, Senate, U.S. Congressional, and School Board districts.</p>
```

**Review by steveoh — CHANGES_REQUESTED** [review]
I know it's not related to what you've done here but since we're changing this page will you update all the old references to 2012. I'm not sure who all missed those changes.

File: data/political/2022-2032-house-senate-congressional-districts/index.html (hunk 1 above)
Comment (rkelson) [reply data/political/2022-2032-house-senate-congressional-districts/index.html:161]: What references do you want changed on the 2012 page?

Why would we make this suggested change? The name of the layer is _Utah District Combination Areas 2022_. I did not create it. It probably should have be created with the new naming convention but the cat's out of the bag now.

**Comment by steveoh** [comment]
We link to the renamed document in this post. The build needs to pass.
 
> Could not find document 'data/political/2012-2021-house-senate-congressional-districts/index.html' in tag 'link'. Make sure the document exists and the path is correct. in /opt/build/repo/_posts/2012-05-02-new-utah-political-districts-and-voting-precincts-gis-datasets.md

**Review by steveoh — APPROVED** [review]

**Pull request merged**